backend/tournaments/tools.py

```diff
@@ -72,35 +72,66 @@ def get_tournament_buckets(tournament):
 
   Return:
     List of dictionaries [{'name': <tournament name>,
-                           'members': [<username>, ...]}, ...]
+                           'members': [{
+													    'name': <player name>,
```

Comment:
Something with indentation here. Probably you use tabs for indenting. Use space instead.